### PR TITLE
Marine Vending Cleanup and Organization

### PR DESCRIPTION
**Pull request by Alterist**
## About The Pull Request
Organizes lists of spawnables
Places masks at the end of the list of  vendor options since it's the least used option
Removes Regular helmets and PAS vests from Medic, Engineer and Leader's regular/Jaeger loadout
Adds PAS vest to armor options and the appropriate helmet to helmet options
gets rid of extra text in SL baldr light option
moves combat backpack to the top of medic pointbuy instead of in between pill options
## Why It's Good For The Game
less shit spawning and pixel hunting to click on stuff, QoL I guess

## Changelog
:cl:
del: No more regular helmets for medics, engies, and leaders when they choose jaeger loadout.
qol: Cleanup and organizes marine vending file
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -1273,14 +1263,14 @@ GLOBAL_LIST_INIT(available_specialist_sets, list("Scout Set", "Sniper Set", "Dem
 		/obj/item/clothing/glasses/hud/health,
 		/obj/item/clothing/under/marine/corpsman,
 		/obj/item/clothing/shoes/marine/full,
+		/obj/item/clothing/head/helmet/marine/corpsman,
```
Comment (SplinterGP) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:1266]: duplicated
```suggestion
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 1 above)
Comment (Moggram) [reply code/game/objects/machinery/vending/new_marine_vendors.dm:1266]: @SplinterGP  My bad, I stated a review some time ago, but forgot to hit submit. I'm not trying to step on your toes.

**Review by SplinterGP — CHANGES_REQUESTED** [review]
i think you did some borking on your PR.

File: code/game/objects/machinery/vending/new_marine_vendors.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Alterist) [reply code/game/objects/machinery/vending/new_marine_vendors.dm]: Done

Comment (Alterist) [reply code/game/objects/machinery/vending/new_marine_vendors.dm]: Fixed

**Comment by Moggram** [comment]
ummmm... what the hell is going on?

**Comment by Alterist** [comment]
i fucked up and fixed it (hopefully)

**Comment by BraveMole** [comment]
Seems a bit useless since tabbed vendor is coming, but a fix is a fix, so why not

**Comment by Chaznoodles** [comment]
Move the machete from the clothing vendor to the weapons vendor if you fancy it please, has always bugged me.

**Comment by Alterist** [comment]
@Chaznoodles that's not within the purview of this PR, this is only changing the automated marine loadout vendors

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**